static/resources/serviceworker.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC
+// Copyright 2020 Mozilla Corporation
```

Comment:
Can we undo this change?  This is the only change in this file -- I'm guessing that this was the wrong file.